spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -54,48 +127,70 @@
       Base64.strict_decode64(seeds[:security_key_direct][:credential_creation_options][:challenge])
     end
 
-    let(:origin) { "http://localhost:3000" }
+    context "when there is a single origin" do
+      include_context "with a single origin"
 
-    let(:attestation_response) do
-      response = seeds[:security_key_direct][:authenticator_attestation_response]
+      let(:origin) { "http://localhost:3000" }
```

Comment:
Maybe I'm being a little bit nitpicky but to be honest I find these shared examples making it a little bit confusing and less readable – for example we have like three ways of overriding what we are setting inside the context:

- The one highlighted here:

```
      include_context "with a single origin"

      let(:origin) { "http://localhost:3000" }
```

then:

```
include_context "with rp_id set to", "localhost"
```


and

```
      include_context "with multiple origins" do
        let(:allowed_origins) do
          [
            seeds[:android_key_direct][:origin],
            "android:apk-key-hash:blablablablablalblalla",
            "localhost"
          ]
        end
      end
```

It feels to me that, in order to avoid blocking this PR while we attempt to iteratively improve the readability of this file, maybe we can go with the more verbose option of avoiding shared examples and contexts for the moment and then take another pass at this file in an attempt to avoid so much repetition as part of different effort. What do you think?